### ISSUE TITLE
Web component: Masthead L1 - On iPhone Voiceover, screen reader users are not able to navigate through the options within the combo box

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#4775]
## Detailed description

Issue: On iPhone Voiceover, screen reader users are not able to navigate through the options within the combo box

Expected: The user should be able to navigate through the options  using left and right swipe and make the selection.

**Comment by ariellalgilmore** [comment]
@Praveen-sr Can this be re-tested? Voiceover looks sounds good on my IPhone when using left and right swipe

**Comment by Praveen-sr** [comment]
@ariellalgilmore Ram re tested this, Below is the expected behavior for the Edit combo box in iPhone Voiceover: 
As we provide the input, Screen reader should provide a feedback announcing the number of suggestions which are displayed so that the user would know that the options are available. If there are no match found, Screen reader should announce “No results”. 
After providing the input, user should be able to Close the Keypad by pressing “done” button. After that,
Should be able to navigate through the suggestions using Left/Right swipe and select one.
Please refer the Search combo box in apple.com website which seems have the similar behavior.
https://www.apple.com/in/

**Comment by Praveen-sr** [comment]
@ariellalgilmore Ram re tested this on [link](https://ibmdotcom-webcomponents.s3.us-east.cloud-object-storage.appdomain.cloud/deploy-previews/4875/index.html.), issue is Partially fixed.
Now Voiceover announces the list of options which are displayed as we type the input. However, When we try to navigate through the options after activating the "done" button in the Keypad, the options disappeared. Hence we are unable to Access them. 

As we mentioned earlier, Below is the expected behavior for the Edit combo box in iPhone Voiceover: 
As we provide the input, Screen reader should provide a feedback announcing the number of suggestions which are displayed so that the user would know that the options are available. If there are no match found, Screen reader should announce “No results”. 
After providing the input, user should be able to Close the Keypad by pressing “done” button. 
After that, should be able to navigate through the suggestions using Left/Right swipe and select one.
Please refer the Search combo box in apple.com website which seems have the similar behavior.
https://www.apple.com/in/

**Comment by RobertaJHahn** [comment]
Reviewed with Ram.

**Comment by RobertaJHahn** [comment]
Ram asked for this suggested results to work the same way it works in the Local Modal.  The top suggestions are read out.  Ram will provide a recording of the behavior he is suggesting for Ariella.

**Comment by Praveen-sr** [comment]
@ariellalgilmore Please find the recording in the [box](https://ibm.ent.box.com/folder/100897002419)

**Comment by jeffchew** [comment]
Related React issue: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/3585

**Comment by RobertaJHahn** [comment]
Moving to icebox because a technical solution is not available at this time. Search would have to be completely reworked / designed in order to find a solution.

**Comment by andy-blum** [comment]
I'm unable to replicate this issue at this time - has the L1 changed enough that this issue no longer exists?
